src/types.ts

```diff
@@ -64,8 +65,10 @@ export interface TSchedule {
 }
 
 export interface THomeButton {
-  text: string;
-  icon: React.ReactElement<SvgIconProps>;
-  to: string;
   color: string;
+  href?: string;
```

Comment:
is this just for the links thats open in a new window?

Reply:
Yes, but I [separated to two types](https://github.com/CodeForBoulder/upswyng/pull/87/commits/528bc8c126123fbf3e9a90fd9540612e9e192f19).